polymetis/polymetis/python/polysim/envs/mujoco_manipulator.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Tuple, List
+import logging
+import mujoco
+import numpy as np
+import os
+
+from omegaconf import DictConfig
+
+from polysim.envs import AbstractControlledEnv
+from polymetis.utils.data_dir import get_full_path_to_urdf
+
+log = logging.getLogger(__name__)
+
+
+class MujocoManipulatorEnv(AbstractControlledEnv):
+    def __init__(
+        self,
+        robot_model_cfg: DictConfig,
+        gui: bool = False,
+        gui_width: int = 1200,
+        gui_height: int = 900,
+        use_grav_comp: bool = True,
+        gravity: float = 9.81,
+    ):
+        self.robot_model_cfg = robot_model_cfg
+        self.robot_description_path = get_full_path_to_urdf(
+            self.robot_model_cfg.robot_description_path
+        )
+        robot_desc_mjcf_path = (
+            os.path.splitext(self.robot_description_path)[0] + ".mjcf"
+        )
+        assert os.path.exists(
+            robot_desc_mjcf_path
+        ), f"No MJCF file found. Create an MJCF file at {robot_desc_mjcf_path} to use the MuJoCo simulator."
```

Comment:
This is a reasonable solution for now -- just make sure to document the need for both urdf and mjcf. The logic is not immediately apparent that you need both.